packages/toolpad-app/src/toolpadDataSources/googleSheets/types.ts

```diff
@@ -55,7 +55,43 @@ export type GoogleSpreadsheet = {
   values: any[][];
 };
 
-export type GoogleSheet = {
+export type GoogleDriveFiles = {
```

Comment:
Does the Google Drive API export a type for this that we can use? Because if they ever change their response in some version, it could break undetected.

Reply:
Yes, I can potentially import the types exported by `@googleapis`. Would it make sense to do that for all the types then?

Reply:
I don't know, which other types? It could also make sense to just pick out of it only the things we are interested in. e.g. `Pick<GoogleDriveApi.Files, 'id' | 'name' | '...'>`